recipes/h5py/bld.bat

```diff
@@ -0,0 +1,4 @@
+SET HDF5_DIR="%PREFIX%"
```

Comment:
You are probably correct. But I guess it won't be %PREFIX%
Maybe  %LIBRARY_PREFIX%?